client/webserver/site/src/js/markets.ts

```diff
@@ -1703,7 +1703,10 @@ export default class MarketsPage extends BasePage {
       else rateStr = Doc.formatRateFullPrecision(ord.rate, market.baseUnitInfo, market.quoteUnitInfo, cfg.ratestep)
       details.rate.textContent = mord.header.rate.textContent = rateStr
       header.baseSymbol.textContent = market.baseUnitInfo.conventional.unit
-      details.type.textContent = market.quoteUnitInfo.conventional.unit
+
+      details.type.textContent = 'Fred'
```

Comment:
testing?...

Reply:
yup .. now I need to find the right replacement variable for the order type .. my first time in the UI code